modules/oe_whitelabel_paragraphs/oe_whitelabel_paragraphs.module

```diff
@@ -68,6 +68,12 @@ function oe_whitelabel_paragraphs_theme_suggestions_field_alter(array &$suggesti
  * Implements hook_preprocess_paragraph().
  */
 function oe_whitelabel_preprocess_paragraph__oe_links_block(array &$variables): void {
+  /** @var \Drupal\oe_whitelabel\ColorSchemePreprocess $color_scheme_preprocess */
+  $color_scheme_preprocess = \Drupal::classResolver(ColorSchemePreprocess::class);
```

Comment:
I've updated the list of paragraphs that recive or not the color scheme and have or not background color to match the requirements